app/src/main/java/com/example/campusguide/directions/ChooseDesitnationOptions.kt

```diff
@@ -0,0 +1,51 @@
+package com.example.campusguide.directions
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.location.Location
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.fragment.app.DialogFragment
+import com.example.campusguide.R
+
+class ChooseDestinationOptions(private val locationSelectedListener: (location: Location) -> Unit): DialogFragment() {
```

Comment:
There's a typo in the file name itself

Reply:
So that's why it wasn't showing up as a class in the project structure 👍